From 6432be8786a28f4168d7f4d3a72766f1d1219592 Mon Sep 17 00:00:00 2001
From: Marsell Kukuljevic <marsell@joyent.com>
Date: Wed, 1 Feb 2017 23:57:06 +0800
Subject: [PATCH] PUBAPI-1360: Test failing due to improved error message from
 fwapi Reviewed by: Cody Mello <cody.mello@joyent.com> Reviewed by: Pedro P.
 Candel <pedro@joyent.com>

---
 test/machines/firewall-rules.js | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/test/machines/firewall-rules.js b/test/machines/firewall-rules.js
index 85f03d6..90f0719 100644
--- a/test/machines/firewall-rules.js
+++ b/test/machines/firewall-rules.js
@@ -122,7 +122,12 @@ module.exports = function (suite, client, other, machine, callback) {
             t.deepEqual(body, {
                 code: 'InvalidParameters',
                 message: 'Invalid parameters',
-                errors: []
+                errors: [ {
+                    code: 'InvalidParameter',
+                    field: 'rule',
+                    message: 'Subnet "10.99.99.0/23" is invalid (bits set to ' +
+                        'right of mask)'
+                } ]
             });
 
             t.end();
-- 
2.21.0

